sadcowpng/sharedspaces/views.py

```diff
@@ -20,5 +26,101 @@ def sign_up(request):
     return render(request, 'sharedspaces/signup.html')
 
 
+# Need to add data fields that auto populate using authentication - will be done in models
```

Comment:
Is this something that still needs to be done? Again, comments should always reflect the final decision that was made, not uncertainty

Reply:
That will be added later in the next iteration.